src/index.js

```diff
@@ -181,7 +181,10 @@ export class Tonic extends window.HTMLElement {
           return this._prop(o)
         }
         case '[object Object]':
-        case '[object Function]': return this._prop(o)
+        case '[object Function]':
+        case "[object Set]":
+        case "[object Map]":
+        case "[object WeakMap]": return this._prop(o)
```

Comment:
lets use single quotes everywhere

Reply:
ah my bad! didn't save on formatting in my editor because it caused all kinds of wild changes too

not sure if this would've been caught with lint, but running lint didn't do anything 🤔

```
> standard src/index.js

standard: Use JavaScript Standard Style (https://standardjs.com)
standard: Some warnings are present which will be errors in the next version (https://standardjs.com)
  /Users/andrewchou/GitHub/achou11/tonic/src/index.js:0:0: File ignored because of a matching ignore pattern. Use "--no-ignore" to override. (undefined) (warning)
```

Reply:
you can replace `index.js` by `./index.js` in the `.gitignore` in this PR

Reply:
I will fix ignore files myself